packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1431,15 +1432,22 @@ class StackViewport extends Viewport implements IStackViewport {
    */
   public async setStack(
     imageIds: Array<string>,
-    currentImageIdIndex = 0,
-    retrieveConfiguration: IRetrieveConfiguration
+    currentImageIdIndex = 0
   ): Promise<string> {
     this._throwIfDestroyed();
 
     this.imageIds = imageIds;
     this.currentImageIdIndex = currentImageIdIndex;
     this.targetImageIdIndex = currentImageIdIndex;
-    this.retrieveConfiguration = retrieveConfiguration || this;
+    const imageRetrieveConfiguration = metaData.get(
```

Comment:
this is odd, since metadata providers only have query and imageId, what is 'stack'?